docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -81,14 +81,10 @@ function extractSlots(options: {
     filePath: filename,
     project,
     checkDeclarations: true,
-    shouldResolveObject: ({ name }) => {
-      // TODO v7: Remove the `components` fallback once `slots` is used everywhere
-      return name === 'slots' || name === 'components';
-    },
+    shouldResolveObject: ({ name }) => name === 'slots',
     shouldInclude: ({ name, depth }) => {
       // The keys allowed in the `components` prop have depth=2
```

Comment:
Could have probably updated this comment to `slots` as well. 🤔

Reply:
#11098